### PR TITLE
Add `yaml_option` support to starting_items

### PR DESCRIPTION
**Pull request by silasary**
Literally the same syntax as categories.json

### CODE REVIEW

File: schemas/Manual.game.schema.json [hunk 1]
```diff
@@ -80,9 +80,18 @@
                     "description": "(Optional) how many items of this block will be randomly added to inventory. Will add every item in the block if not included",
                     "type":"integer"
                 },
+                "yaml_option": {
+                    "description": "(Optional) Array of Options that will decide if the items & locations in this category are enabled",
```
Comment (nicopop) [inline schemas/Manual.game.schema.json:84]:
```suggestion
                    "description": "(Optional) Array of Options that will decide if this block is rolled",
```

**Review by nicopop — CHANGES_REQUESTED** [review]
other than this small definition, looks good

File: schemas/Manual.game.schema.json (hunk 1 above)
Comment (nicopop) [reply schemas/Manual.game.schema.json:84]: oops forgot to add the why for this sugested change:
in startinv if you look in the other defs you can see that its starting_items's blocks we define not items & locations

**Review by FuzzyGamesOn — DISMISSED** [review]

**Comment by FuzzyGamesOn** [comment]
Looks fine. Just needs the merge conflict resolved 😃

**Review by FuzzyGamesOn — APPROVED** [review]

**Review by nicopop — APPROVED** [review]

**Pull request merged**